packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenu.md

```diff
@@ -25,9 +26,7 @@ class SingleOption extends React.Component {
       super(props);
       this.state = {
         isOpen: false,
-        toggleTemplateProps: {
-          text: "Options menu"
-        },
+        toggleTemplateText: "Options menu",
```

Comment:
Don't think the toggleTemplateText should be a state variable in the examples

Reply:
Probably not, but we aren't consistent across examples anyway.